graphql_compiler/cost_estimation/statistics.py

```diff
@@ -1,10 +1,28 @@
 # Copyright 2019-present Kensho Technologies, LLC.
 from abc import ABCMeta, abstractmethod
+from dataclasses import dataclass
 import datetime
+import math
+from typing import Any, Dict, List, Optional, Tuple
 
 import six
 
 
+@dataclass
+class SamplingSummary:
+    """Results extracted from sampling a random rowset that are relevant for statistics."""
+
+    # The class this summary describes
+    class_name: str
```

Comment:
Let's try to be consistent between `class/type/vertex` as much as possible. I'm not sure which one we've been using the most, but I feel like it's more than one.

Reply:
Yeah, `class/type/vertex/location_name`... it's not great. This is actually specific to a vertex so I'll rename to 'vertex'. Edge sampling summaries will be much different.